assets/models/system/roberta-base/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/roberta-base/bc2764f/mlflow_model_folder
+  container_path: huggingface/roberta-base/1694430215/mlflow_model_folder
```

Comment:
Could you please help to add following information in the description: 

1. Online endpoint created and a snapshot of request/response
2. Run to save model to blobstorage

Reply:
conda.yaml
channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.3.1
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.3.1
  - azureml-core==1.52.0
  - azureml-mlflow==1.52.0
  - azureml-metrics==0.0.24
  - scikit-learn==1.1.0
  - cryptography==41.0.2
  - python-dateutil==2.8.2
  - datasets==2.14.4
  - soundfile==0.12.1
  - librosa==0.10.0.post2
  - diffusers==0.19.3
  - sentencepiece==0.1.99
  - transformers==4.31.0
  - torch==2.0.0
  - accelerate==0.21.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.24
- pycocotools=2.0.4
name: mlflow-env

-------------------------------
MLmodel
flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForMaskedLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: roberta-base
    model_data: data
    pytorch_version: 2.0.0+cu117
    task_type: fill-mask
    transformers_version: 4.31.0
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.3.1
model_uuid: 6ad6da3e6a1e49be9170fe5f61c6cd02
utc_time_created: '2023-08-31 05:46:10.103230'